src/ansys_sphinx_theme/layout.html

```diff
@@ -24,6 +24,12 @@
     {% endif %}
 {% endblock %}
 
+
+{% block extrahead %}
+    {% include 'announcement.html' %}
```

Comment:
As per previous comments:

```suggestion
    {% include 'announcement_layout.html' %}
```